backend/corpora/dataset_processing/process.py

```diff
@@ -430,21 +411,50 @@ def main():
         )
         update_db(dataset_id, processing_status=status)
 
-    # Process cxg
-    process_cxg(local_filename, dataset_id, os.environ["CELLXGENE_BUCKET"])
 
-    # Process metadata
-    metadata = extract_metadata(local_filename)
-    logger.info(metadata, flush=True)
-    update_db(dataset_id, metadata)
+def main():
+    check_env()
+    dataset_id = os.environ["DATASET_ID"]
+    try:
```

Comment:
combine this try block with the one bellow. This well reduce code duplication in the error handling.